projects/common/src/navigation/navigation.service.ts

```diff
@@ -304,6 +307,7 @@ export interface InAppNavigationParams {
   queryParams?: QueryParamObject;
   queryParamsHandling?: 'merge' | 'preserve';
   relativeTo?: ActivatedRoute;
+  skipLocationChange?: boolean;
```

Comment:
this is already captured on line 306 (`replaceCurrentHistory`)

Reply:
Got it, thanks, will use it instead adding new